Src/DependencyCollector/Shared/HttpDependenciesParsingTelemetryInitializer.cs

```diff
@@ -96,5 +98,49 @@ private bool TryParseAzureBlob(string host, string name, string url, out string
 
             return result;
         }
+
+        private bool TryParseAzureTable(string host, string name, string url, out string account, out string verb, out string tableName)
+        {
+            bool result = false;
+
+            account = null;
+            verb = null;
+            tableName = null;
+
+            if (name != null && host != null && url != null)
+            {
+                if (host.EndsWith("table.core.windows.net", StringComparison.OrdinalIgnoreCase))
+                {
+                    ////
+                    //// Table Service REST API: https://msdn.microsoft.com/en-us/library/azure/dd179423.aspx
+                    ////
+
+                    account = host.Substring(0, host.IndexOf('.'));
+
+                    string nameWithoutVerb = name;
+
+                    for (int i = 0; i < this.azureTableVerbPrefixes.Length; i++)
+                    {
+                        var verbPrefix = this.azureTableVerbPrefixes[i];
+                        if (name.StartsWith(verbPrefix, StringComparison.OrdinalIgnoreCase))
+                        {
+                            verb = name.Substring(0, verbPrefix.Length);
+                            nameWithoutVerb = name.Substring(verbPrefix.Length);
```

Comment:
Why are the names prefixed with verbs like "GET"?
What happens if someone names their table something like "GettingDataTable" and you strip off the "Get" portion of the name?